### PR TITLE
UI: Align the `icon` in the `Alert`

### PR DESCRIPTION
**Pull request by lstng**
**What is this feature?**
Aligning the icon inside the Alert component in the `ui` package.

**Why do we need this feature?**

In the case of the multi-line, we are not changing anything.
However, for the one line scenario, the icon was a bit too low. 

**Storybook**
_Before_
bug

_After_ (with just title, or with no title, and  just a child)
after

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by ashharrison90 — COMMENTED** [review]
so this misalignment problem still exists when there is child content, it's just way less obvious because of the extra text.

i think the easiest thing to do is honestly just adjust the icon position up 1px. you could modify `styles.icon` to just have `position: relative; top: -1px;`, add a comment that this is to align with the title text, and that would be it 🤷

**Comment by joannaWebDev** [comment]
> so this misalignment problem still exists when there is child content, it's just way less obvious because of the extra text.
> 
> i think the easiest thing to do is honestly just adjust the icon position up 1px. you could modify `styles.icon` to just have `position: relative; top: -1px;`, add a comment that this is to align with the title text, and that would be it 🤷

Thank you for the suggestion @ashharrison90 . I agree is more straightforward.

File: packages/grafana-ui/src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -150,8 +149,11 @@ const getStyles = (
         zIndex: -1,
       },
     }),
+    iconWrapper: css({ alignSelf: 'center' }),
```
Comment (ashharrison90) [inline packages/grafana-ui/src/components/Alert/Alert.tsx:152]: this can be removed 👍

**Review by ashharrison90 — APPROVED** [review]
unused style can be removed, otherwise lgtm 👍

**Pull request merged**